app/models/like.rb

```diff
@@ -1,3 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :author
+  belongs_to :user, foreign_key: :author_id
+  belongs_to :post
+
+  after_save :update_likes_counter
```

Comment:
- [ ] Great job-making use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) :100: